packages/core/src/inspector.ts

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 
-import { instantiateBaseInteractor, findElements, unsafeSyncResolveParent } from "./constructor";
-import { BaseInteractor, Filters, InteractorConstructor, FilterParams } from "./specification";
+import { findElements, unsafeSyncResolveParent, instantiateInteractor } from "./constructor";
+import { Interactor, Filters, InteractorConstructor, FilterParams } from "./specification";
 
 type GetElement<I extends InteractorConstructor<any, any, any, any>> = I extends InteractorConstructor<infer E, any, any, any> ? E : never
 type GetFilters<I extends InteractorConstructor<any, any, any, any>> = I extends InteractorConstructor<any, infer F, any, any> ? F : never
 type GetActions<I extends InteractorConstructor<any, any, any, any>> = I extends InteractorConstructor<any, any, any, infer A> ? A : never
 
-export interface InteractorInspector<E extends Element, FP extends FilterParams<any, any>> extends BaseInteractor<E, FP> {
+export interface InteractorInspector<E extends Element, FP extends FilterParams<any, any>> extends Omit<Interactor<E, FP>, 'find'> {
```

Comment:
How do you feel about naming this `InspectorConstructor`, that seems a bit more consistent to me?

I could also see just inlining the relevant methods from `Interactor` here, rather than using `Omit`. But I think that's a matter of taste.

I think we should omit `exists` and `absent` as well here, right?

Reply:
@wKich ☝🏻

Reply:
I don't think so about renaming it to `InspectorConstructor`, because `all` method of `Inspector` instantiates interactors for each element.

I use Omit because I need to redefine `find` method with a different signature

We can omit `exists` and `absent` as well as `is` and `has`. But why we should do it?

Reply:
Makes sense to me!